docs/TutorEnvironmentManager.rst

```diff
@@ -0,0 +1,108 @@
+Tutor Environment Manager
+#########################
+
+Tutor environment manager allows you to create one project and set one version for it.
+
+You will be able to continue using the `tutor version manager`_ commands in your project:
+
+- tvm list
+- tvm plugins list
+
+
+Creating a project
+------------------
+
+To create a new project you should call ``tvm project init``, this configures a new tvm project in the current path and set your-project as ``TUTOR_ROOT`` and ``TUTOR_PLUGINS_ROOT``.  
+
+You can create different projects with the same tutor version, you have 2 options:
+
+- Use the global active version.
+
+.. code-block:: bash  
+    
+    # Using the global active version
+    
+    mkdir your-project && cd your-project
+    tvm project init
+    
+- Use a specific installed version, this creates a new folder with the ``project-name`` to host it.
+
+.. code-block:: bash  
+    
+    # Specify the version
+    
+    tvm project init project-name v14.0.0
+    
+
+
+Activate / Deactivate a project
+-------------------------------
+
+To activate a project you should run
```

Comment:
Maybe we need to specify that this command will active a virtualenv with python, pip, and tutor.